### PR TITLE
TL Translations PR #1

### PR DESCRIPTION
**Pull request by severinolorillajr**
Hi @markstory ,

This is the first batch of Tagalog translations.
All of these undergo a Proofreading and review, to ensure good quality.
I already have a 3 developers working on this document, and a lot more to come.
After each review, I personally made one last check and suggest some revisions if needed.
In the next few days/weeks, some other developers will also work on this project, once we are done with one of our ongoing projects.

Thanks,
Sevs

### CODE REVIEW

File: tl/console-and-shells/completion-shell.rst [hunk 1]
```diff
@@ -1,91 +1,91 @@
-Completion Shell
+Pagkumpleto ng Shell
 ################
```
Comment (markstory) [inline tl/console-and-shells/completion-shell.rst]: Title underlines need to match the title length. For example this should be:

```rst
Pagkumpleto ng Shell
####################
```

When title underlines are too short, sphinx fails to create headlines correctly.

**Comment by severinolorillajr** [comment]
Hi @markstory ,
Thanks for the feedback.
Already fixed the headers that caused the building failure.
Let me know if I missed somethings.
Cheers,
Sev

File: tl/console-and-shells/completion-shell.rst [hunk 2]
```diff
@@ -1,91 +1,91 @@
...
-Create a file named **cake** in **/etc/bash_completion.d/** and put the
-:ref:`bash-completion-file-content` inside it.
+Lumikha ng file na pinangalanang **cake** sa **/etc/bash_completion.d/** at ilagay ang 
+:ref:`bash-completion-file-content` sa loob nito.
 
-Save the file, then restart your console.
+I-save ang file, pagkatapas ay i-restart ang iyong console.
 
-.. note::
+.. tandaan::
```
Comment (markstory) [inline tl/console-and-shells/completion-shell.rst]: `.. note::` is a directive for sphinx and should not be translated. `.. warning::` is also a directive that should not be translated.

File: tl/console-and-shells/cron-jobs.rst [hunk 3]
```diff
@@ -16,25 +16,25 @@ trivial to setup, for example::
...
+---------------------------------
 
-On some shared hostings ``cd /full/path/to/root && bin/cake myshell myparam``
-might not work. Instead you can use
+Sa ilang mga shared na hosting ``cd /full/path/to/root && bin/cake myshell myparam``
+ay maaaring hindi gumana. Sa halip maaari mong gamitin ang 
 ``php /full/path/to/root/bin/cake.php myshell myparam``.
 
-.. note::
+.. tandaan::
```
Comment (markstory) [inline tl/console-and-shells/cron-jobs.rst]: Same here.

**Pull request merged**